mlflow/azureml/__init__.py

```diff
@@ -237,7 +237,7 @@ def build_image(
         return image, registered_model
 
 
-@experimental
+@deprecated("the azureml deployment plugin", since="1.19.0")
```

Comment:
Can we include a plugin URL in the message?

Reply:
Updated with a short url to our docs